task_sdk/tests/api/test_client.py

```diff
@@ -82,6 +83,47 @@ def make_client(transport: httpx.MockTransport) -> Client:
     return Client(base_url="test://server", token="", transport=transport)
 
 
+class TestTaskInstanceOperations:
+    """
+    Test that the TestVariableOperations class works as expected. While the operations are simple, it
+    still catches the basic functionality of the client for task instances including endpoint and
+    response parsing.
+    """
+
+    # TODO: Add tests for different ti endpoints
```

Comment:
I am handling this part through a different PR: https://github.com/apache/airflow/pull/44723